src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#" class="logo">
+        <img src="images/logo.png"
+        alt="Moyo logo"
+        class="logo_image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link__apple is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
 
          <li class="nav__item">
            <a href="#" class="nav__link">Smartphones</a>
          </li>
```

add blank line between multiple sibling blocks.